SECURITY.md

```diff
@@ -1,35 +1,25 @@
 # Security Policy
 
-## Reporting a Vulnerability
+## Reporting a Security Issue
 
-[Responsible disclosure](https://en.wikipedia.org/wiki/Responsible_disclosure) enhances security for the entire community.
-
-### Keybase
-
-We prefer [keybase](https://keybase.io) as the vehicle for reporting security issues as that gives us an encrypted forum to discuss, analyze, and remediate the threat before an exploit is published.
-Please request to join the [typelevelsec](https://keybase.io/team/typelevelsec) team, and we will create a subteam to discuss the issue with you.
-
-### E-mail
-
-E-mail is not encrypted, but is preferred to a zero-day!
-If you cannot use Keybase, please email [security@typelevel.org](mailto:security@typelevel.org).
+To report a security issue, please email [security@typelevel.org](mailto:security@typelevel.org) with a description of the issue, the steps you took to create the issue, affected versions, and, if known, mitigations for the issue.  The Security Team will attempt to respond within 3 working days of your email.  If the issue is confirmed as a vulnerability, we will open a Security Advisory.  This project follows a 90 day disclosure timeline.
```

Comment:
A public key for the team would be in line with Rust, Debian, and Mozilla.  Typelevel has a 1Password for sharing the private key.  I already have one sitting locally for this purpose.  Let me know.